packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -9,13 +9,27 @@ import useLogout from './useLogout';
 import useNotify from '../sideEffect/useNotify';
 import { AuthProvider } from '../types';
 
-jest.mock('./useLogout');
-jest.mock('../sideEffect/useNotify');
+let loggedIn = true;
 
-const logout = jest.fn();
```

Comment:
this was a bad idea and made me loose 1 hour scratching my head. Using authProvider.logout() in useLogout is way more natural.